libcxx/lib/abi/CHANGELOG.TXT

```diff
@@ -16,6 +16,13 @@ New entries should be added directly below the "Version" header.
 Version 20.0
 ------------
 
+* [libcxx][libc] Implements from_chars floating-point
+
+  All platforms
+  -------------
+  Symbol added: _ZNSt6__ndk127__from_chars_floating_pointEPKcS1_RdNS_12chars_formatE
+  Symbol added: _ZNSt6__ndk127__from_chars_floating_pointEPKcS1_RfNS_12chars_formatE
```

Comment:
These symbols seem a bit wrong: it looks like they are coming from the Android CI jobs, is that possible? Android uses a special namespace instead of `__1`, so it would be better to take the name generated from another Linux job since that represents better the name of the symbols being added.